<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="imageview" name="graphicsView"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="path_to_image">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="path_to_image_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="open_image_button"/>
    <addaction name="exit_button"/>
    <addaction name="new_choose"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="zoom_in_button"/>
    <addaction name="zoom_out_button"/>
    <addaction name="fit_button"/>
    <addaction name="fit_inv_button"/>
    <addaction name="choose_area"/>
    <addaction name="change_view"/>
    <addaction name="start_processing"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Просмотр</string>
    </property>
    <addaction name="show_orig"/>
    <addaction name="show_result"/>
    <addaction name="show_final"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="open_image_button"/>
   <addaction name="start_processing"/>
   <addaction name="change_view"/>
   <addaction name="zoom_in_button"/>
   <addaction name="zoom_out_button"/>
   <addaction name="fit_button"/>
   <addaction name="fit_inv_button"/>
   <addaction name="separator"/>
   <addaction name="show_orig"/>
   <addaction name="show_result"/>
   <addaction name="show_final"/>
   <addaction name="separator"/>
   <addaction name="exit_button"/>
  </widget>
  <action name="open_image_button">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pref/icons/open-file.png</normaloff>:/pref/icons/open-file.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="toolTip">
    <string>Открыть изображение для обработки</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="exit_button">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pref/icons/exit.png</normaloff>:/pref/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>Выход из программы</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="zoom_in_button">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pref/icons/zoom_in.png</normaloff>:/pref/icons/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Масштаб +</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="zoom_out_button">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pref/icons/zoom_out.png</normaloff>:/pref/icons/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Масштаб -</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="fit_button">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pref/icons/fit_inv.png</normaloff>:/pref/icons/fit_inv.png</iconset>
   </property>
   <property name="text">
    <string>Масштаб по размеру экрана</string>
   </property>
   <property name="toolTip">
    <string>Масштаб по размеру экрана</string>
   </property>
  </action>
  <action name="fit_inv_button">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pref/icons/fit.png</normaloff>:/pref/icons/fit.png</iconset>
   </property>
   <property name="text">
    <string>Фактический размер</string>
   </property>
  </action>
  <action name="choose_area">
   <property name="text">
    <string>Выбрать область</string>
   </property>
  </action>
  <action name="change_view">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pref/icons/Artboard_19-512.png</normaloff>:/pref/icons/Artboard_19-512.png</iconset>
   </property>
   <property name="text">
    <string>Переключить вид</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="new_choose">
   <property name="text">
    <string>new_choose</string>
   </property>
  </action>
  <action name="start_processing">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pref/icons/1746178-200.png</normaloff>:/pref/icons/1746178-200.png</iconset>
   </property>
   <property name="text">
    <string>Начать обработку</string>
   </property>
  </action>
  <action name="show_orig">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pref/icons/orig.png</normaloff>:/pref/icons/orig.png</iconset>
   </property>
   <property name="text">
    <string>Показать оригинал</string>
   </property>
  </action>
  <action name="show_result">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pref/icons/res.png</normaloff>:/pref/icons/res.png</iconset>
   </property>
   <property name="text">
    <string>Показать результат</string>
   </property>
  </action>
  <action name="show_final">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pref/icons/plus.png</normaloff>:/pref/icons/plus.png</iconset>
   </property>
   <property name="text">
    <string>Показать наложение</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>imageview</class>
   <extends>QGraphicsView</extends>
   <header>imageview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
